roles/neutronstar/tasks/main.yml

```diff
@@ -39,14 +52,20 @@
   lineinfile:
      dest: "{{ ngi_pipeline_conf }}/{{ bash_env_sthlm_script }}"
      line: >
-         alias neutronstar='nextflow run {{ neutronstar_dest }}/main.nf -profile standard,uppmax
-         -c {{ ngi_pipeline_conf }}/neutronstar_{{ item.site }}.config'
+         alias neutronstar='nextflow run {{ neutronstar_dest }}/main.nf -profile uppmax \
+         -c {{ ngi_pipeline_conf }}/nextflow_irma_{{ item.site }}.config \
+         -c {{ ngi_pipeline_conf }}/neutronstar_{{ item.site }}.config --supernova_container {{ supernova_container_path }}'
```

Comment:
Cool 👍 

We could move `supernova_container` in to the neutronstar config file, but seeing as we will hopefully get rid of this config file before long, I think it's better how you currently have it ✅

Reply:
👍